src/config/plugin.ts

```diff
@@ -136,7 +136,13 @@ export class Plugin implements IPlugin {
 
   constructor(public options: PluginOptions) {}
 
-  public async load(): Promise<void> {
+  /**
+   * Loads a plugin
+   * @param isWritingManifest - if true, exclude selected data from manifest
+   * default is false to maintain backwards compatibility
+   * @returns Promise<void>
+   */
+  public async load(isWritingManifest?: boolean): Promise<void> {
```

Comment:
Have you considered adding `isWritingManifest` as a new field in the `options` object here?
https://github.com/oclif/core/blob/main/src/config/plugin.ts#L137

That way we could load plugins in `oclif manifest` like this:
```ts
const plugin = new Plugin({root: fullPath, isWritingManifest: true})
await plugin.load()
```

instead of adding a new arg to `Plugin.load`